app/assets/javascripts/Components/submission_file_manager.jsx

```diff
@@ -127,6 +132,47 @@ class SubmissionFileManager extends React.Component {
     this.setState({showModal: true, uploadTarget: uploadTarget})
   };
 
+  showFile = (item) => {
```

Comment:
Rename this to `updateViewFile` (makes it more obvious that this is a state action, not a rendering one).